Readme.md

```diff
@@ -73,6 +73,16 @@ chaintest.generator.chainlp.enabled=true
 chaintest.generator.chainlp.host.url=<host:port>
 ```
 
+## Is Docker must to generate ChainTest Report?
+No, you do not necessarily need Docker to generate and view static reports with ChainTest. Docker is a convenient way to deploy and run applications with all their dependencies in an isolated environment, but for generating static reports, it is not a strict requirement.
+
+## When Docker is Needed for ChainTest?
+Docker is useful in these cases:
```

Comment:
Would suggest a small change here:

~Docker is useful in these cases:~

In ChainTest's context, Docker is required only if setting up ChainLP given one or more of the below requirements: